package.json

```diff
@@ -43,7 +43,7 @@
   },
   "homepage": "https://github.com/3box/3box-js#readme",
   "dependencies": {
-    "3box-orbitdb-plugins": "^1.0.5",
+    "3box-orbitdb-plugins": "3box/3box-orbitdb-plugins#feat/orbit-23",
```

Comment:
Should we release this before merging this PR?

Reply:
will change in release PR instead, since few days out